arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -9,163 +9,163 @@
 		<array>
 			<dict>
 				<key>displayName</key>
-				<string>Display a map</string>
+				<string>Display map</string>
 				<key>storyboardName</key>
 				<string>DisplayMap</string>
 				<key>descriptionText</key>
-				<string>This sample shows you how to display a map with imagery basemap</string>
+				<string>Display a map with an imagery basemap.</string>
 			</dict>
 			<dict>
 				<key>displayName</key>
 				<string>Change map view background</string>
 				<key>storyboardName</key>
 				<string>ChangeMapViewBackground</string>
 				<key>descriptionText</key>
-				<string>This sample shows you how to customize map view&apos;s background grid</string>
+				<string>Customize map view&apos;s background by changing its grid properties.</string>
 			</dict>
 			<dict>
 				<key>displayName</key>
-				<string>Map loaded</string>
+				<string>Map load status</string>
 				<key>storyboardName</key>
 				<string>MapLoaded</string>
 				<key>descriptionText</key>
-				<string>This sample shows how to tell what the map&apos;s load status is</string>
+				<string>Determine the map&apos;s load status.</string>
 			</dict>
 			<dict>
 				<key>displayName</key>
-				<string>Open map (URL)</string>
+				<string>Open map URL</string>
 				<key>storyboardName</key>
 				<string>OpenMapURL</string>
 				<key>descriptionText</key>
-				<string>This sample shows you how to open a map using its url</string>
+				<string>Display a web map.</string>
 			</dict>
 			<dict>
 				<key>displayName</key>
 				<string>Change basemap</string>
 				<key>storyboardName</key>
 				<string>SwitchBasemap</string>
 				<key>descriptionText</key>
-				<string>Changing the basemap is too easy. This sample shows you how it can be done</string>
+				<string>Change a map&apos;s basemap.</string>
 			</dict>
 			<dict>
 				<key>displayName</key>
-				<string>Set initial map area</string>
+				<string>Map initial extent</string>
```

Comment:
Yesterday I asked Mike and others and was a bit confused about this. 😕 
Mike indicated we should stick to the design doc during this phase for README clean up. In the design doc the title is `Map initial extent`, whereas clearly it is better to keep the original title as `Set initial map area`. I just cannot be sure which one I should use...  

@vquach2404 What's your thought?

Reply:
Although iOS may differ from the common design, I think we should keep the original title for this README update phase and leave all title changes for a separate effort. Since it may include standardizing the titles amongst all SDKs.

Reply:
Uh-oh... 😨  I already let in 1 change for title here: https://github.com/Esri/arcgis-runtime-samples-ios/commit/2a99336f7284846723fed35934bb61fd202825ec

Should I revert it? I thought it was OK to change the title both in README and in ContentPlist, but just now realized that we shouldn't change the title in this sprint.

Reply:
Current practice:

For the `Map initial extent` or `Set initial map area` sample, I kept the original title as `Set initial map area`.

There was a change in a previous PR https://github.com/Esri/arcgis-runtime-samples-ios/commit/2a99336f7284846723fed35934bb61fd202825ec. Please share your thoughts on whether I should revert it or not.

Reply:
I suppose it would be nice to revert them so that all sample renames happen at the same time.